spec/framesSpec.js

```diff
@@ -0,0 +1,65 @@
+describe('roll', function() {
+  let frame = new Frame;
```

Comment:
could maybe have used a before each for your Frame objects, but the semantics of the object names is useful to the readability of the tests so I dunno which I would prefer. 🤷